quic/s2n-quic-core/events/connection.rs

```diff
@@ -198,6 +205,7 @@ struct ConnectionStarted<'a> {
 //= https://tools.ietf.org/id/draft-marx-qlog-event-definitions-quic-h3-02#5.1.3
 /// Connection closed
 struct ConnectionClosed {
+    #[nominal_counter("error", "")]
```

Comment:
should we drop the unit? it's not clear to me when there's ever an appropriate non-empty unit on counters?

Reply:
yeah that makes sense. i'll fix it